src/common/low_precision_transformations/tests/gather_transformation.cpp

```diff
@@ -0,0 +1,219 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "layer_transformation.hpp"
+
+#include <string>
+#include <sstream>
+#include <memory>
+
+#include <gtest/gtest.h>
+
+#include <transformations/utils/utils.hpp>
+#include <transformations/init_node_info.hpp>
+#include <low_precision/gather.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+#include "lpt_ngraph_functions/common/dequantization_operations.hpp"
+#include "lpt_ngraph_functions/gather_function.hpp"
+#include "simple_low_precision_transformer.hpp"
+
+namespace {
+using namespace testing;
+using namespace ngraph::pass;
+using namespace ngraph;
+
+class GatherTransformationTestValues {
+public:
+    class Actual {
+    public:
+        ngraph::element::Type precisionBeforeDequantization;
+        ngraph::builder::subgraph::DequantizationOperations dequantization;
+    };
+
+    class Expected {
+    public:
+        ngraph::element::Type precisionBeforeDequantization;
+        ngraph::builder::subgraph::DequantizationOperations dequantizationBefore;
+        ngraph::element::Type precisionAfterOperation;
+        ngraph::builder::subgraph::DequantizationOperations dequantizationAfter;
+    };
+
+    std::vector<int> gatherIndicesValues;
+    std::vector<int> axis;
+    TestTransformationParams params;
+    Actual actual;
```

Comment:
opset8::Gather operation has also `batch_dims` attribute. Can we cover this by tests?

Reply:
Done. Thanks!